js/count-letter-in-string.test.js

```diff
@@ -0,0 +1,7 @@
+const count = require('./count-letter-in-string');
+
+describe('Count letter in string', () => {
+  test('If get string aba should return object a:2 b:1', () => {
```

Comment:
For such cases, how about just naming this test case:

> should work correctly for a simple string

Instead of putting all the details in the test case's name 🙂 ? The details can be easily read from the test's code